packages/venia-concept/src/components/Header/header.css

```diff
@@ -0,0 +1,33 @@
+main .header-animation {
+    background: var(--venia-global-submenu);
+    animation-name: fadeInDown;
+    animation-direction: normal;
+    animation-play-state: running;
+    animation-fill-mode: forwards;
+    animation-iteration-count: 1;
+    animation-duration: .7s;
+    animation-delay: 0s;
+    animation-timing-function: ease-out;
+}
+
+@-webkit-keyframes fadeInDown {
+    0% {
+        opacity: 0;
+        -webkit-transform: translateY(-20px);
+    }
+    100% {
+        opacity: 1;
+        -webkit-transform: translateY(0);
+    }
+}
+
+@keyframes fadeInDown {
+    0% {
+        opacity: 0;
+        transform: translateY(-20px);
+    }
+    100% {
+        opacity: 1;
+        transform: translateY(0);
+    }
+}
```

Comment:
I don't think that we need this separate file
We can move it to the header.module.css